.gitignore

```diff
@@ -5,4 +5,7 @@
 .vscode
 .env
 .DS_Store
-.tableland*
\ No newline at end of file
+.tableland*
+coverage/
+tableland-artifacts/
```

Comment:
Is this here because of local-tableland? With the version you're using that should no longer be getting created.

Reply:
oops, yup that's leftover, will remove.